app/services/async_measurements_creator.rb

```diff
@@ -0,0 +1,23 @@
+class AsyncMeasurementsCreator
+  SLICE_SIZE = 500
+  AMOUNT_THRESHOLD = 20_000
```

Comment:
I guess just `threshold` is enough? And also amount is uncountable, so it's better to use `number_threshold` if needed. Check out this for reference: https://goo.gl/images/HQGoZe 😄